backend/README.md

```diff
@@ -38,7 +38,9 @@ will be cached and it will not take so long.
 12. Ctrl + D to get out of psql, Ctrl + D again to get out of the postgres container.
 13. Finally, `docker-compose down` to take down the postgres container and `docker-compose up` to start running the whole application.
 
-## Setup: Basic Search API
+<span style="color:red">Please note there are significant changes to Search API. Rerun Step 3 to make sure new views are created in your local database. The materialized view creation will take few minutes to complete.</span>
```

Comment:
Remove span and style. The color is not showing up. I don't think it is possible to color plain text in Github markdown.